src/pages/test-report/cat-a-mod1/components/speed-check-modal/speed-check-modal.ts

```diff
@@ -0,0 +1,30 @@
+import { Component } from '@angular/core';
+import { IonicPage, NavParams, ViewController } from 'ionic-angular';
+import { ModalEvent } from '../../../test-report.constants';
+import { speedCheckLabels } from '../../../../../shared/constants/competencies/cata-mod1-competencies';
+
+@IonicPage()
+@Component({
+  selector: 'speed-check-modal',
+  templateUrl: 'speed-check-modal.html',
+})
+export class SpeedCheckModal {
+
+  speedCheck: speedCheckLabels[];
```

Comment:
`speedChecksNeedCompleting` for example would be a better name for this property